src/Generator/Commands/TestCases/ApiTestCaseGenerator.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Apiato\Core\Generator\Commands\TestCases;
+
+use Apiato\Core\Generator\FileGeneratorCommand;
+
+class ApiTestCaseGenerator extends FileGeneratorCommand
+{
+    protected string|null $fileName = 'ApiTestCase';
+
+    public static function getCommandName(): string
+    {
+        return 'apiato:generate:testcase:functional:api';
```

Comment:
refactor to use `make` instead of `generate`

Reply:
Done in de2ba4c2e6127447ab7e4eff8306eda135f73cfa